albatross.ml

```diff
@@ -585,17 +620,20 @@ module Make (S : Tcpip.Stack.V4V6) = struct
                 (fun trie (name, p) -> fst (Vmm_trie.insert name p trie))
                 Vmm_trie.empty ps
             in
-            let state = { error = None; policies; configuration } in
+            let state = { status = Online; policies; configuration } in
             Lwt.return state
         | Ok w ->
-            Logs.warn (fun m ->
-                m "init: unexpected policy reply for %s: %a"
-                  (Configuration.name_to_str configuration.name)
-                  (Vmm_commands.pp_wire ~verbose:false)
-                  w);
+            let err =
+              Fmt.str "Unexpected policy reply for %s: %a"
+                (Configuration.name_to_str configuration.name)
+                (Vmm_commands.pp_wire ~verbose:false)
+                w
+            in
+            Logs.warn (fun m -> m "%s" err);
+            let error = Status.make `Internal err in
```

Comment:
It is a bit unclear to me what `` `Internal `` actually means -- since the issue here is that the remote albatross replied with something unexpected --> this means either Mollymawk needs to be updated or downgraded (or the remote albatross needs to be downgraded or updated) --> since the remote administrator may influence the behaviour, I lean towards making this transient, but as said, I'm not sure.